src/scenes/Lines/scenes/Editor/JourneyPatterns/Editor/index.tsx

```diff
@@ -51,6 +56,24 @@ const JourneyPatternEditor = ({
     setDirectionSelection(directionSelection);
   };
 
+  const deleteStopPoint = (index: number) => {
```

Comment:
Dette er altså den viktigste endringen: ved sletting av stoppepunkt sletter jeg også samme indeks i passeringstidene for hver serviceJourney.